app/assets/javascripts/components/overview/overview_handler.jsx

```diff
@@ -47,13 +50,29 @@ const Overview = createReactClass({
 
   componentDidMount() {
     ServerActions.fetch('timeline', this.props.course_id);
+    this.timeout = this.poll();
     return ServerActions.fetch('tags', this.props.course_id);
   },
 
+  componentWillUnmount() {
+    if (this.timeout) {
+      clearInterval(this.timeout);
+    }
+  },
+
   storeDidChange() {
     return this.setState(getState());
   },
 
+  poll() {
+    if (this.state.course.type === 'Editathon') {
+      return setInterval(() => CourseActions.updateCourse(), POLL_INTERVAL);
```

Comment:
I don't see how this could have worked for you. CourseActions.updateCourse() does not fetch the course, it just updates the store when the user makes changes to to the course. Calling it with no data is basically a noop.

I had to change it to `ServerActions.fetch('course', this.state.course.slug)`.